lib/openapi_contracts/validators/documented.rb

```diff
@@ -19,7 +19,7 @@ def request_missing
     end
 
     def response_desc
-      "#{request.request_method} #{request.path}"
+      "#{response.request.request_method} #{response.request.path}"
```

Comment:
I think this was a merge error. The original is correct (directly calling request).

Reply:
Yes, exactly, it was a merge error. But I fixed it in the last commit together with the Rubocop issue. I think you are commenting on an outdated change. In the `Files changed` tab it doesn't show that change: https://github.com/mkon/openapi_contracts/blob/861974c65320f395b34a3376e0ca34588e139ec6/lib/openapi_contracts/validators/documented.rb#L22

Reply:
ah yes i looked at one commit only 👍